clustered/client/src/main/java/org/ehcache/clustered/client/service/ClusteringService.java

```diff
@@ -17,14 +17,47 @@
 package org.ehcache.clustered.client.service;
 
 import org.ehcache.clustered.client.config.ClusteringServiceConfiguration;
+import org.ehcache.clustered.client.internal.store.ServerStoreProxy;
+import org.ehcache.core.spi.store.Store;
 import org.ehcache.spi.service.PersistableResourceService;
+import org.ehcache.spi.service.ServiceConfiguration;
 
 /**
- * @author Clifford W. Johnson
+ * Provides support for accessing server-based resources.
  */
 public interface ClusteringService extends PersistableResourceService {
 
   ClusteringServiceConfiguration getConfiguration();
 
+  /**
+   * Gets a {@link ServerStoreProxy} though which a server-resident {@code ServerStore} is accessed.
+   *
+   * @param cacheIdentifier the {@code ClusteredCacheIdentifier} for the cache for which a
+   *                        {@code ServerStoreProxy} is requested
+   * @param storeConfig the configuration used for the {@link Store} for which the {@code ServerStoreProxy}
+   *                    is requested
+   * @param <K> the cache-exposed key type
+   * @param <V> the cache-exposed value type
+   *
+   * @return a new {@code ServerStoreProxy}
+   */
+  <K, V> ServerStoreProxy<K, V> getServerStoreProxy(ClusteredCacheIdentifier cacheIdentifier, final Store.Configuration<K, V> storeConfig);
```

Comment:
Or maybe there is just a naming issue. `ClusteringService` being to general for the role this interface has.